benchmark/utils.go

```diff
@@ -55,3 +56,55 @@ func GetImageListFromCsv(csvLoc string) ([]ImageDescriptor, error) {
 	}
 	return images, nil
 }
+
+func GetCommitHash() (string, error) {
+	cmd := exec.Command("git", "rev-parse", "HEAD")
+	output, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+	return string(output), nil
+}
+
+func GetDefaultWorkloads() []ImageDescriptor {
+	var imageList []ImageDescriptor
+	imageList = append(imageList,
+		ImageDescriptor{
```

Comment:
While we're here, can we rename this? "image descriptor" already means something pretty specific in the container ecosystem.

Reply:
Are we using ImageDescriptor somewhere else in SOCI?